src/util/operations.tsx

```diff
@@ -5,10 +5,14 @@ export const getInstanceName = (operation?: LxdOperation): string => {
   // /1.0/instances/<instance_name>
   // /1.0/instances/<instance_name>?project=<project_name>
   // /1.0/instances/<instance_name>/snapshots/<snapshot_name>
+  // /1.0/instances/<instance_name>/<snapshot_name>
   return (
     operation?.resources?.instances
-      ?.filter((item) => item.startsWith("/1.0/instances/"))
-      .map((item) => item.split("/")[3])
+      ?.filter((item) => {
+        // for snapshots backend returns %2F instead of /, which is not decoded by the browser
+        return decodeURIComponent(item).startsWith("/1.0/instances/");
+      })
+      .map((item) => decodeURIComponent(item).split("/")[3])
```

Comment:
I think this is the wrong way to do it, now it reports to be creating an instance with the source instance name. I think we can filter out items with either more than 3 `/` or with `%2F` in it. Then the instance list and operation list should link to the correct/new instance name.

Reply:
aaah I was filtering out any resource url with more than 4 segments if split by `/`. But then decided to not do that and failed to see the problem with the source instance name. Will revert to filtering out resources.

Reply:
I think those tests are relevant. I.e. when you create a snapshot, you can see an operation in the operation list. And that operation should link to the correct instance. Maybe my suggestion to filter out the snapshot URLs wasn't the best idea here. There should be a solution that works for snapshot creation. Keeping the operation list link to the correct instance in that case. As well as for the instance and operation list linking to the new instance, when creating an instance from snapshot.

Reply:
The snapshot creation would be linked to the correct instance using the current approach. Since `resources.instances` contains the source instance for the snapshot?

Reply:
I see, then the tests seem to be wrong. I was wondering maybe an older version did report the resources differently. But even with 5.0 I did get the instances / instances_snapshots structure you mentioned above. So sounds good to me to remove those tests and change the logic.

Can you please report this as a bug to LXD? I think it is wrong, that the instances list includes a link to the snapshot when creating an instance from a snapshot.

Reply:
Sure will send a message to the UI channel :+1: